### ISSUE TITLE
Move link profile to context menu next to hide/delete profile

### ISSUE THREAD

**Issue by VitGottwald** [zowe/vscode-extension-for-zowe#765]
The profile node in ds/uss/job tree is getting crowdy.
<img width="261" alt="Screenshot 2020-04-30 at 10 56 45" src="https://user-images.githubusercontent.com/19686979/80703579-4f3f9e00-8ae3-11ea-8fdb-5f3c10e9b1ed.png">

I believe that only often used functionality should have an icon in there and would suggest moving the _Link Profile_ (Associate Profiles) next into context menu next to Hide/Delete Profile 

<img width="151" alt="Screenshot 2020-04-30 at 10 36 47" src="https://user-images.githubusercontent.com/19686979/80703700-8ca42b80-8ae3-11ea-9279-e3bd2eb6d23e.png">

**Comment by jelaplan** [comment]
@VitGottwald 
I agree with your suggestion. I thought we had informally agreed to keep these button bars to three icons. This doesn't seem like a common activity. 

I'm wondering about this feature/icon with the tooltip 'Associate Profile.' Is there another story that describes the use case and rationale? Maybe 631? I guess the idea is that a user could create a z/OSMF profile with the + icon and then switch it to a ssh profile. I'm wondering if this is meant to be the workflow for allowing connections to services other than z/OSMF. 
<img width="1011" alt="image" src="https://user-images.githubusercontent.com/3207260/81229505-e9a95100-8fbd-11ea-8339-ec55a94a7dfc.png">

**Comment by jelaplan** [comment]
It seems this may be related to your issue 734 around associated profiles icon. 
https://github.com/zowe/vscode-extension-for-zowe/issues/734
